src/App.tsx

```diff
@@ -1,61 +1,131 @@
+import { useState, ChangeEvent } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number) {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  // if there is no user with a given userId
+  return foundUser || null;
+}
+
+export const tasks = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [taskUser, setTaskUser] = useState('0');
+  const [task, setTask] = useState('');
```

Comment:
`task` is an object, right? 
You can't set a string as a default value to something that should contain an object. Default value should be empty object or, if you want to emphasise that it is empty, you may choose undefined.